Sources/Turf/Geometries/LineString.swift

```diff
@@ -215,4 +215,114 @@ extension LineString {
         
         return closestCoordinate
     }
+
+    private func getSquareDistance(coordinate1: CLLocationCoordinate2D, coordinate2: CLLocationCoordinate2D) -> Double {
+        let dx = coordinate1.longitude - coordinate2.longitude
+        let dy = coordinate1.latitude - coordinate2.latitude
+        return dx * dx + dy * dy
+  }
+
+    private func simplifyRadialDistance(_ coordinates: [CLLocationCoordinate2D], tolerance: Double) -> [CLLocationCoordinate2D] {
```

Comment:
The first argument isn’t a distance, so it needs an argument label. How would we describe this method in a sentence?

Reply:
Tried out a new name that I think makes sense. Also moved the function to be public since it might be generally useful.